src/RadMarshak/test_radiation_marshak.cpp

```diff
@@ -34,6 +34,7 @@ template <> struct quokka::EOS_Traits<SuOlsonProblem> {
 	static constexpr double mean_molecular_mass = 1.0;
 	static constexpr double boltzmann_constant = 1.0;
 	static constexpr double gamma = 5. / 3.;
+	static constexpr double hydrogen_mass_code_units = 1.0 / C::n_A;
```

Comment:
Now that we've updated Microphysics to the post-2019 SI definitions, `1/N_A` should not be used anywhere.